tedana/combine.py

```diff
@@ -137,7 +136,9 @@ def make_optcom(data, tes, adaptive_mask, t2s=None, combmode='t2s', verbose=True
     tes : (E,) :obj:`numpy.ndarray`
         Array of TEs, in seconds.
     adaptive_mask : (S,) :obj:`numpy.ndarray`
-        Adaptive mask of the data indicating the number of echos with signal at each voxel
+        Adaptive mask of the data indicating the number of echos with signal at each voxel.
+        This mask may be thresholded, typically with values less than 3 set to 0 depending
```

Comment:
Docstring suggestion for clarity :point_down: 

```suggestion
        This mask may be thresholded; for example, with values less than 3 set to 0.
        For more information on thresholding, see `make_adaptive_mask`.
```